easybuild/easyconfigs/b/BiasAdjustCXX/BiasAdjustCXX-1.9.1-foss-2021b.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'BiasAdjustCXX'
+version = '1.9.1'
+
+homepage = "https://biasadjustcxx.readthedocs.io/en/latest/"
+
+description = """
+BiasAdjustCXX command-line tool for the application of fast and efficient bias corrections in climatic research
+"""
+
+# Note: A compiler bug(?) in template deduction prevents newer versions of icpc to compile this software.
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/btschwertfeger/BiasAdjustCXX/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9f2988d0b1676c57037cdb5cbb4e313b31c4fc08a1ea8a55ea5eccc5a2bc5861']
+
+dependencies = [
+    ('netCDF', '4.8.1'),
+    ('CDO', '2.0.5'),
+    ('NCO', '5.0.3'),
+]
```

you're missing a build dependency on `CMake` here
```suggestion
builddependencies = [
    ('CMake', '3.22.1'),
]

dependencies = [
    ('netCDF', '4.8.1'),
    ('CDO', '2.0.5'),
    ('NCO', '5.0.3'),
]
```